jlc_kicad_tools/jlc_lib/cpl_fix_rotations.py

```diff
@@ -18,7 +18,7 @@
 import csv
 import re
 import sys
-from jlc_kicad_tools.logger import Log
+import logging
```

Comment:
Why did you remove our custom logging?

Reply:
Sorry, i did not see an option like that. My start was at your commit 16c30db8c1a07f47bacc4fccce14b1db3205a048
I also did not delete anything exept the tabs, which i replace by more pythonic spaces.